src/core/Parcel.ts

```diff
@@ -136,14 +136,17 @@ export class Parcel {
 
     public toJSON() {
         const { nonce, fee, networkId, action } = this;
-        if (!nonce || !fee) {
+        if (!fee) {
             throw Error("Nonce and fee in the parcel must be present");
```

Comment:
It seems the error message should be changed also

Reply:
done